tools/gbench/report.py

```diff
@@ -126,16 +128,15 @@ def find_test(name):
            if 'real_time' in bn and 'cpu_time' in bn)
     for bn in gen:
         fmt_str = "{}{:<{}s}{endc}{}{:+16.4f}{endc}{}{:+16.4f}{endc}{:14.0f}{:14.0f}{endc}{:14.0f}{:14.0f}"
-        special_str = "{}{:<{}s}{endc}{}{:16.4f}{endc}{}{:16.4f}"
+        special_str = "{}{:<{}s}{endc}{}{:16.4f}{endc}{}{:16.4f}{endc}{}          {}"
 
         if last_name is None:
             last_name = bn['name']
         if last_name != bn['name']:
-            MIN_REPETITIONS = 2
-            if ((len(timings_time[0]) >= MIN_REPETITIONS) and
-                (len(timings_time[1]) >= MIN_REPETITIONS) and
-                (len(timings_cpu[0]) >= MIN_REPETITIONS) and
-                    (len(timings_cpu[1]) >= MIN_REPETITIONS)):
+            if ((len(timings_time[0]) >= UTEST_MIN_REPETITIONS) and
+                (len(timings_time[1]) >= UTEST_MIN_REPETITIONS) and
+                (len(timings_cpu[0]) >= UTEST_MIN_REPETITIONS) and
+                    (len(timings_cpu[1]) >= UTEST_MIN_REPETITIONS)):
```

Comment:
weird indent here.. is this correct?

Reply:
This is correct, but the indent is indeed wrong.

Reply:
Funny, `autopep8` actually insists on this weird indent.

Reply:
as long as it does the right thing :)